packages/menu/src/MenuItem.tsx

```diff
@@ -143,12 +143,10 @@ interface SharedMenuItemProps {
    * Class name that will be applied to root MenuItem element.
    */
   className?: string;
-
```

Comment:
nit, I prefer the extra white space between the definitions as it gives space for my eyes to rest before reading the next prop name.

Having no line break makes it feel like a run-on sentence.

Reply:
That being said, I don't feel strong enough to desire a code change, but just adding my opinion into the mix.

Reply:
Definitely agree actually, and I've made those changes. Things changed as a result of copy/pasting when trying to make sure I used the same descriptions across the typescript, the docstring, and the readme.